hbase-server/src/test/java/org/apache/hadoop/hbase/master/TestMasterShutdown.java

```diff
@@ -129,7 +129,7 @@ public void testMasterShutdownBeforeStartingAnyRegionServer() throws Exception {
     try {
       htu =  new HBaseTestingUtility(
         createMasterShutdownBeforeStartingAnyRegionServerConfiguration());
-
+      htu.getConfiguration().setInt("hbase.client.retries.number", 64);
```

Comment:
Any further context on this value? Smaller values were still causing test to fail?